wordpress-mobile/WordPress-iOS
# Stories: 16.8 Bug Fixes

Related Kanvas PR: https://github.com/tumblr/kanvas-ios/pull/93

- Fixes a bug where the Media Picker could be rotated to landscape and mess up the UI. Notice the size of the view controller + tab bar and the "Add" button in the "before" video below:

https://user-images.githubusercontent.com/3250/109680694-fe049e80-7b39-11eb-91be-a903ef451ce7.mp4

- Fixes a bug when scaling Portrait video to fill the device screen. Note that this does slow down video import a little.

| Before | After |
| ------ | ----- |
| <a href="https://user-images.githubusercontent.com/3250/109605557-94a57100-7ae2-11eb-92e6-2aaa024100d0.PNG"><img src="https://user-images.githubusercontent.com/3250/109605557-94a57100-7ae2-11eb-92e6-2aaa024100d0.PNG" width="300"></a> | <a href="https://user-images.githubusercontent.com/3250/109605589-9e2ed900-7ae2-11eb-9372-c07dd938b3b1.PNG"><img src="https://user-images.githubusercontent.com/3250/109605589-9e2ed900-7ae2-11eb-9372-c07dd938b3b1.PNG" width="300"></a> |

- Disables Stories on iPad for the 16.8 release.

### Testing
* Open the Story Editor.
* Open the Media Picker using the button in the bottom left.
* Rotate the device to landscape and ensure that the status bar remains fixed.
* Select a **video** which was taken in **Portrait** by your iPhone.
* Ensure that the video appears in the correct aspect ratio and is not warped.

#### iPad
* Tap on the FAB from the Blog Details or Post list screens
* Ensure that the "Story post" option is not available on iPad
* When attempting to edit a story, ensure that a message appears when tapping the edit button.
* The Story block should not be available in the block picker when adding a new block.

PR submission checklist:
